src/test/java/core/basesyntax/service/ReportServiceTest.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class ReportServiceTest {
+    private static final String BREAK = System.lineSeparator();
+    private static final String HEADER = "fruit,quantity" + BREAK;
+    private static ReportService reportService;
+    private static StorageDaoImpl storageDao;
+
+    @BeforeAll
+    static void setUp() {
+        storageDao = new StorageDaoImpl();
+        reportService = new ReportService(storageDao);
+    }
+
+    @Test
+    void createReport_validOutput_Ok() {

```

Comment:
```suggestion
    void createReport_validOutput_ok() {
```
fix naming everywhere, we should be consistent - use lowercase after underscore